src/util/models.py

```diff
@@ -11,7 +11,12 @@
 
 from .const import ENVIRON
 from .fields import IMD_FIELD_PATTERN, remove_field
-from .helpers import _ir_values_value, _remove_export_lines, _validate_model, model_of_table, table_of_model
+from .helpers import (
+    _ir_values_value,
+    _validate_model,
+    model_of_table,
+    table_of_model,
```

Comment:
Could be kept in one line
(remove the trailing comma to force `ruff` to reformat it on one line)
```suggestion
    table_of_model
```